hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -280,9 +280,10 @@ static QString* get_var(struct PixelShader *ps, int reg, bool is_dest)
         return qstring_from_str("r1");
     case PS_REGISTER_V1R0_SUM:
         add_var_ref(ps, "r0");
-        return qstring_from_str("(v1 + r0)");
+        return qstring_from_str("vec4(v1.xyz + r0.xyz, 0.0)");
```

Comment:
nit: this should be `.rgb`

Reply:
`v1.rgb` or `r0.rgb` or both?

Reply:
Given that this belongs to register combiners: everything.

The register combiners only deal with colors, in particular only RGB [`.rgb`], or only ALPHA [`.a`].
It makes no sense to ever use `.xyzw` in this code section; everything should be `.rgba` (ideally split: `.rgb` + `.a`)